deploy.md

```diff
@@ -1,59 +1,149 @@
-Here are the instructions to deploy Activity Stream:
+# Overview
 
-#  Have the correct credentials
-# Update code and packages
+At the end of each milestone, we will start a new release process that consists of following tasks.
```

Comment:
@ncloudioj Yes exactly.  The release process begins when we snapshot master into a release branch, which happens at the beginning of each new milestone.  The snapshotted release branch will then have the changes from the previous milestone in it.